miro_preprocessor/xml_to_json_converter/run.py

```diff
@@ -30,19 +30,32 @@
 from utils import generate_images
 
 
-def main(bucket, src_key, dst_key):
+def main(bucket, src_key, dst_key, js_path="json"):
     image_data = generate_images(bucket=bucket, key=src_key)
 
     tmp_json = tempfile.mktemp()
     os.makedirs(os.path.dirname(tmp_json), exist_ok=True)
+
+    s3 = boto3.client('s3')
+
     with open(tmp_json, 'w') as f:
         for img in image_data:
+            img_json_dump = json.dumps(img, separators=(',', ':'))
```

Comment:
Because we’re making assertions about the exact output in the tests, I’d add `sort_keys=True` to the parameters to `json.dumps()` – that ensures we have a stable sort order in the JSON. Otherwise you’re relying on dict iteration order, which isn’t guaranteed..